### DISCUSSION TITLE
Not able to change the color of selected date and its range

### DISCUSSION THREAD

**Discussion by BraveEvidence** [gpbl/react-day-picker#2605] [Support]
Thanks for reporting a bug for DayPicker. Please provide here a brief description of the issue you’ve encountered. 

## To reproduce

https://codesandbox.io/p/sandbox/daypicker-bug-report-forked-yqxffx

Here is the code

```tsx
import { DayPicker } from "react-day-picker";
import "react-day-picker/style.css";

export default function App() {
  return  <DayPicker
            mode="range"
            disabled={{
              after: new Date(),
            }}
          />;
}
```

## Actual Behavior

I don't know how to change the color of the date which has been picked as well as the range which has been picked

## Expected Behavior

Should be able to change  color of the date which has been picked as well as the range which has been picked

## Screenshots
<img width="327" alt="Screenshot 2024-11-25 at 7 17 14 PM" src="https://github.com/user-attachments/assets/752d1ef9-514d-4f01-b630-63121a8d52ba">

Check the above screenshot, the background color of the selected date is blue and the range is light blue, how to change this color?

**Comment by gpbl** [comment]
@BraveEvidence thanks for your feedback! 

You can override the `--rdp-range_middle-background-color` CSS variables in the `.rdp-root`:

```css
/* ./styles.css */
.rdp-root {
  --rdp-range_middle-background-color: orange;
}
```

in your JavaScript file, make sure you import the custom styles after the `react-day-picker/styles.css`:

```ts
import { DayPicker } from "react-day-picker";

// Make sure react-day-picker/style.css is before your custom style
import "react-day-picker/style.css";

import "./styles.css"; // Include custom styles
```

See this codesandbox: 
https://codesandbox.io/p/sandbox/daypicker-bug-report-forked-vfjklj?file=%2Fsrc%2FApp.tsx%3A6%2C34&workspaceId=c9c7c415-3883-483b-9f1c-8a70ae6a6363

<img width="329" alt="Screenshot 2024-11-26 at 6 27 09 PM" src="https://github.com/user-attachments/assets/c79de20e-4b7d-422f-86cf-51f285aebc94">

**Comment by BraveEvidence** [comment]
@gpbl  Thanks for the reply, what about the blue color for the selected date, any ideas?

**Comment by vitorschelb** [comment]
After Hours trying, that the only way it worked for me:
            modifiersStyles={{
              range_start: {
                backgroundColor: '#3182CE',
                color: 'white',
                borderRadius: '50%',
                boxShadow: '10px 0px 0px #354A53',
                position: 'relative',
                zIndex: 2
              },
              range_end: {
                backgroundColor: '#3182CE',
                color: 'white',
                borderRadius: '50%',
                boxShadow: '-10px 0px 0px #354A53',
                position: 'relative',
                zIndex: 2 
              },
              range_middle: {
                backgroundColor: '#354A53',
                boxShadow: '-10px 0px 0px #354A53, 10px 0px 0px #354A53',
                position: 'relative',
                zIndex: 1 
              }
            }}
